src/components/StudentCredentialPage.jsx

```diff
@@ -121,6 +123,7 @@ class StudentCredentialPage extends Component {
           </div>
         );
       } else if (this.props.studentData && this.props.isFetched) {
+        this.props.setUserTypeAction('student');
```

Comment:
create a constant USER_TYPES = {
STUDENT: 'student',
ADMIN: 'admin'
}
```suggestion
        this.props.setUserTypeAction(USER_TYPES.STUDENT);
```

Reply:
Done.